reference/wishlists.v3.yml

```diff
@@ -92,9 +111,7 @@ paths:
               type:
                 type: string
     get:
-      description: >-
-        Returns a list of Wishlists. Optional filter parameters can be passed
-        in.
+      description: Returns a list of wishlists. Optional filter parameters can be passed in.
       summary: Get All Wishlists
```

Comment:
```suggestion
      summary: Get all wishlists
```

Reply:
Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.